docs/development/STYLE_GUIDE.md

```diff
@@ -13,6 +13,7 @@ This style guide is an evolving document and will be updated as the VEDA UI proj
 ### Naming for Branches
 
 - Use the related GitHub issue number; if not available, use `hotfix` or `update`.
+- The ideal pattern would be {issue#}-{some}-{meaningful}-{title}
```

Comment:
Heads up that we also have release branches that are not currently related to issues. We will also use the branch name pattern (`release/*`) to run the Playwright test suite.